homeassistant/components/switch/fritzhome.py

```diff
@@ -0,0 +1,66 @@
+"""
+Support for AVM Fritz!Box fritzhome switch devices.
+
+For more details about this component, please refer to the documentation at
+http://home-assistant.io/components/switch.fritzhome/
+"""
+import logging
+from homeassistant.components.fritzhome import DOMAIN
+from homeassistant.components.switch import (SwitchDevice)
+
+DEPENDENCIES = ['fritzhome']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the Fritzhome switch platform."""
+    device_list = hass.data[DOMAIN]
+
+    devices = []
+    for device in device_list:
+        if device.has_switch:
+            devices.append(FritzhomeSwitch(hass, device))
+
+    add_devices(devices)
```

Comment:
You probably also want to call this with `True` as a second parameter to retrieve the switch's status right on start.